src/Metadata/class-post-builder.php

```diff
@@ -92,13 +96,13 @@ private function build_type(): void {
 		/**
 		 * Filters the JSON-LD @type.
 		 *
-		 * @param array $jsonld_type JSON-LD @type value, default is NewsArticle.
+		 * @param string $jsonld_type JSON-LD @type value, default is NewsArticle.
 		 * @param int $id Post ID.
 		 * @param string $post_type The Post type in WordPress.
 		 *
 		 * @since 2.5.0
 		 */
-		$type = (string) apply_filters( 'wp_parsely_post_type', 'NewsArticle', $this->post->ID, $this->post->post_type );
+		$type = apply_filters( 'wp_parsely_post_type', 'NewsArticle', $this->post->ID, $this->post->post_type );
```

Comment:
Are we certain that `(string)` is not needed here?

Reply:
Personally I am confident because of two reasons.

1. We are treating this variable as string going forward.
2. Our test also indicates that we returns string here (like [this one](https://github.com/Parsely/wp-parsely/blob/fix/phpstan-errors/tests/Integration/OtherTest.php#L156))

If you want I can put it back with an ignore PHPStan comment.